src/e2e/specs/dashboard.spec.ts

```diff
@@ -151,9 +160,9 @@ test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard - Headers @smoke
     });
 
     await dashboardPage.fixedTab.click();
-    expect(page.url()).toMatch(/.*dashboard\/fixed\/?/);
+    expect(page.url()).toMatch(/.*dashboard\/fixed.*/);
```

Comment:
Just to make sure: these changes are to accommodate query params, e.g. for direct linking to the dialog, right?

Reply:
Yes, these changes were made to accommodate for additional slugs or query params in the URL.